starboard/extension/media_settings.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_EXTENSION_MEDIA_SETTINGS_H_
+#define STARBOARD_EXTENSION_MEDIA_SETTINGS_H_
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define kStarboardExtensionMediaSettingsName \
+  "dev.cobalt.extension.MediaSettings"
+
+typedef struct StarboardExtensionMediaSettingsApi {
```

Comment:
We started to have more media setting related extensions, please coordinate with @jasonzhangxx to see if we can consolidate.